docs/source/index.rst

```diff
@@ -24,7 +24,8 @@ Having trouble? We’d like to help!
 - Try the :doc:`faq` – it’s got answers to many common questions.
 
 - Looking for specific information? Try the :ref:`genindex`, :ref:`modindex`,
-  or the detailed :doc:`table of contents <contents>`.
+  :doc:`topics section <topics/index>` or detailed :doc:`table of contents
+  <contents>`.
```

Comment:
Nor is the reference material mentioned. This text was ripped straight from Django's documentation. I don't think we need to mention every section here.